includes/classes/Authentications/WordPressBasicAuth.php

```diff
@@ -79,6 +79,10 @@ public static function credentials_form( $args = array() ) {
 				<a href="#" class="manual-setup-button">
 					<?php esc_html_e( 'Manually Set Up Connection', 'distributor' ); ?>
 				</a>
+				<div class="dt-wizard-status">
+					<span class="spinner is-active"></span>
+					<span><?php esc_html_e( 'Checking the connection..', 'distributor' ); ?></span>
```

Comment:
```suggestion
					<span><?php esc_html_e( 'Checking the connection...', 'distributor' ); ?></span>
```

Should be a full 3-dot ellipsis :)